src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Generic.Tests.cs

```diff
@@ -34,6 +34,15 @@ protected PriorityQueue<TElement, TPriority> CreatePriorityQueue(
             return queue;
         }
 
+        [Theory]
+        [InlineData(1)]
+        [InlineData(100)]
+        public void CreateWithCapacity_EqualsCapacityProperty(int capacity)
```

Comment:
Could you also add a few tests demonstrating the change in capacity after modifications are being made as well as after calling `EnsureCapacity()` and `TrimExcess()`?

Reply:
Added one, should do the trick. There's seemingly no tests at all for EnsureCapacity/TrimExcess here.